src/pages/style.css

```diff
@@ -39,12 +39,16 @@
 :root {
   --mobile-width-breakpoint: 728px;
   --tablet-width-breakpoint: 1000px;
+
   --font-body: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
     Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
     'Segoe UI Symbol';
   --font-display: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI',
     Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
     'Segoe UI Symbol';
+
+  /* TODO: what should this be? */
+  --font-monospace: Menlo;
```

Comment:
For more context, I think we need to be using system monospace fonts? But I'm not sure how to do it.

Reply:
When I was skimming the `hashicorp/design-system-tokens` repo I noticed some tokens for this - https://github.com/hashicorp/design-system-tokens/blob/6406550e15588de305a01edd3b6bcf3a47f4a36b/dist/products/css/tokens.css#L298 . TLDR; `SF Mono, Consolas, Ubuntu Mono, monospace;` looks like the system monospace stack they're using at the DS level ✨

Reply:
I took another peek at [the WIP HDS Product Figma](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/?node-id=1436%3A10434) (also there's the link if you haven't seen it yet!), and I'm seeing that `ui-monospace` is a thing? So here's what I'm going with (a combination of what you've found too):

```
--font-monospace: ui-monospace, Menlo, Consolas, Ubuntu Mono, monospace;
```